api/client/samples/analogous_years/lib/get_transform_data.py

```diff
@@ -28,8 +28,10 @@ def get_data(client, metric_id, item_id, region_id, source_id, frequency_id, sta
                    'start_date': start_date}
     client.add_single_data_series(data_series)
     data = client.get_df()
```

Comment:
here you can use the new functionality to only get one series:

`data = client.get_df(index_by_series=True).loc[[tuple(metric_id,item_id, region_id, 0, source_id, frequency_id)], :]
`

Reply:
I think the documentation about this function here need more detail https://developers.gro-intelligence.com/api.html?highlight=index_by_series . 

I am not sure what performance improvement would I see due to indexing by series, though?

Reply:
It's simply giving a view on the df that only accesses the rows of the particular series and excluding all other rows. In this case since you've just modified to only have one data series I guess it makes no difference but it would have solved the original issue :)